bin/test_python.py

```diff
@@ -1,3 +1,23 @@
+import os
+TEST_SYMPY = os.getenv('TEST_SYMPY', '0')
```

Comment:
Change this to `TEST_SYMPY = os.getenv('TEST_SYMPY', False)`, otherwise the condition will always be true.